lib/opentelemetry_avalanche.ex

```diff
@@ -6,35 +6,47 @@ defmodule OpentelemetryAvalanche do
              |> String.split("<!-- MDOC !-->")
              |> Enum.fetch!(1)
 
-  alias OpenTelemetry.Span
+  alias OpenTelemetry.SemanticConventions
+  alias OpenTelemetry.Tracer
 
+  require SemanticConventions.Trace
   require OpenTelemetry.Tracer
 
-  @tracer_id :opentelemetry_avalanche
-
-  @default_config [
-    span_name: "avalanche.query",
-    trace_query_statement: true,
-    trace_query_params: false,
-    trace_query_error: true
-  ]
+  @tracer_id __MODULE__
+
+  @options_schema NimbleOptions.new!(
+                    span_name: [
+                      type: :string,
+                      default: "avalanche.query",
+                      doc: "The name of the span."
+                    ],
+                    trace_query_statement: [
+                      type: :boolean,
+                      default: true,
+                      doc: "Should trace query statement?"
+                    ],
+                    trace_query_params: [
```

Comment:
maybe we add a comentadjust the doc_string below saying: "be careful with allowing this as you may have sensitive data here"